tests/amr/data/particles/schedule/test_main.cpp

```diff
@@ -57,30 +70,39 @@ TEST_F(ALevelWithDomainParticles, ghostParticleNumberisZeroBeforeScheduleAndCorr
     std::cout << "filling ghosts\n";
     schedule->fillData(0.);
 
+    size_t eq = std::get<dim - 1>(std::forward_as_tuple(3 * 2, 3 * 5 * 4));
```

Comment:
explain the numbers, maybe with variables (i.e. not comments, if possible)

Reply:
![image](https://user-images.githubusercontent.com/30491/82203609-d6727b80-9903-11ea-8eaf-f879f4e9dd54.png)

Reply:
why `Cells{5}`?

Reply:
updated

Reply:
clearer. but sorry I can't find where you get these numbers from.
All I see in the input is that the domain is (0,7) in both directions.

Reply:
```https://github.com/PHAREHUB/PHARE/pull/218/files#diff-681d5e53cf20eda2a6298fe996f65536R16```

every level0 cell is refined so we end up with 4 4*4 patches given smallest/largest patch_size == 4

4*4 ghost_box is 6*6
6*6 patch border cells == 5*4 (or (6*2) + (4*2))